.travis.yml

```diff
@@ -12,8 +22,11 @@ install:
   - chmod +x /tmp/urchin/package/urchin
   - '[ -z "$WITHOUT_CURL" ] || sudo apt-get remove curl -y'
 script:
+  - export CXX=g++-4.8
+  - export CC=gcc-4.8
```

Comment:
You can probably use [`env`](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-.travis.yml) here.